rholang/src/main/scala/coop/rchain/rholang/interpreter/RhoRuntimeSyntax.scala

```diff
@@ -39,9 +39,9 @@ final class RhoRuntimeOps[F[_]: Sync](
       initialPhlo: Cost,
       normalizerEnv: Map[String, Par]
   ): F[EvaluateResult] = {
-    implicit val rand: Blake2b512Random = Blake2b512Random(128)
+    val rand: Blake2b512Random = Blake2b512Random(128)
```

Comment:
Construction of random Blake2b512Random is now repeated on many places. Please define it on Blake2b512Random itself like `Blake2b512Random.genDefault` or `Blake2b512Random.defaultRandom`.